packages/react-composites/src/composites/common/PeoplePaneContent.tsx

```diff
@@ -220,3 +221,10 @@ const createDefaultContextualMenuItems = (
   }
   return menuItems;
 };
+
+const canRemoveParticipants = (adapter: CommonCallAdapter): boolean => {
+  const removeParticipantCapability = adapter.getState().call?.capabilitiesFeature?.capabilities.removeParticipant;
```

Comment:
Will capabilities be changed during the call? If yes, let's call useSelector instead of getState directly
If not, either way works (but it might be still a good practice to call useSelector)

Reply:
It can change. For example, when a ACS user can be changed to Attendee in a teams interop meeting. getState still works but I agree with you that selector is better. I will make a selector in a future refactor PR.

Reply:
Update: I [reverting back to using role](https://github.com/Azure/communication-ui-library/pull/4312/files#diff-f58bdd76c9f8e75269e3e181a242a9e249fa15c77497d9cfddbe0c93f051c295L205) because there is a bug with removeParticipant capability for Rooms